packages/esm-patient-search-app/src/patient-search-result/patient-search-result.component.tsx

```diff
@@ -1,35 +1,43 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import dayjs from 'dayjs';
 import styles from './patient-search-result.scss';
-import { interpolateString, ExtensionSlot, useConfig, navigate } from '@openmrs/esm-framework';
+import { interpolateString, ExtensionSlot, useConfig, navigate, attach, detach } from '@openmrs/esm-framework';
 import { SearchedPatient } from '../types/index';
+import CustomOverflowMenuComponent from '../ui-components/overflow-menu.component';
+import OverflowMenuVertical16 from '@carbon/icons-react/es/overflow-menu--vertical/16';
+import { useTranslation } from 'react-i18next';
 
 const PatientSearchResults: React.FC<PatientSearchResultsProps> = ({ patients, hidePanel }) => {
   const config = useConfig();
+  const { t } = useTranslation();
+
+  useEffect(() => {
```

Comment:
Why this effect? I think it that the extension should always be attached here.

Reply:
(in general, why is this a new slot instead of taking the existing slot name? does it really matter if the action button is shown on the patient chart or on the search?)